EDI/UserInterface/01a_Exports.md

```diff
@@ -0,0 +1,340 @@
+- [File settings tab](#file-settings-tab)
+- [Data sources tab](#data-sources-tab)
+- [Mapping settings tab](#mapping-settings-tab)
+
+## Settings tab   
+
+*DataHub Exporter > Exports > Select export definition > Tab Settings*
+
+![Edit definition](../../Assets/Screenshots/EDI/Operation/ExportDefinitionEdit.png "[Edit definition]")
+
+The *Settings* tab is used to edit the basic settings of the exporter definition.
+
+- [CANCEL]   
+   Click this button to finish the exporter definition creation without saving it. The *Create definition* view is closed.
+
+- [SAVE]   
+   Click this button to save the exporter definition. The *Create definition* view is closed.
+
+For all other fields and functions, see [Create exporter definition](#create-exporter-definition).
+
+
+
+## File settings tab  
+
+*DataHub Exporter > Exports > Select export definition > Tab File settings*
+
+![Edit file settings](../../Assets/Screenshots/EDI/Operation/ExportDefMessageStructure.png "[Edit file settings")
+
+The *File settings* tab is used to specify the EDIFACT message fields and segments that your business partner require for the selected message type. This tab does not appear until you have entered the *Template engine* in the *Settings* tab. 
+
+- *Release*   
+   Edit this drop-down list to select the release of the EDIFACT message type your business partner require.
+
+- *Message type*       
+   Edit this drop-down list to select the EDIFACT message type your business partner require.
+
+- *Reference node path*    
+   In case you define the definition for the line items, enter the *Reference node path* manually. For detailed information, see [Get line item segments](../Operation/04_ManageLineItemsFileStt.md#get-line-item-segments). If you define the definition for the message header, you can ignore this field. 
+
+- ![Get](../../Assets/Icons/Get.png "[Get]")   
+   Click this button to load the specified EDIFACT message structure.
+
+ 
+## Data sources tab   
+
+*DataHub Exporter > Exports > Select export definition > Tab Data sources*
+
+![Data sources tab](../../Assets/Screenshots/EDI/Operation/DataSourcesList.png "[Data sources tab]")
+
+The *Data sources* tab is used both to define the attribute set that you want to use for the attribute set mapping of the EDIFACT message and to do the mapping. This tab does not appear until you have successfully saved both the settings and the file settings in the corresponding tabs.
+
+**Exporter definition sources**   
+
+The *Exporter definition sources* view contains a LIST view that contains all exporter definition sources you want to use to perform the attribute mapping. 
+
+- ![Change sorting](../../Assets/Icons/Sort02.png "[Change sorting]") (Change sorting)   
+   Click this button change the sorting of the data sources. The button is only available if the list contains more than one entry. The *Data sources (Sorting mode)* view is displayed, see [Data sources &ndash; Sorting mode](#data-sources-–-sorting-mode).
```

Comment:
```suggestion
   Click this button to change the sorting of the data sources. The button is only available if the list contains more than one entry. The *Data sources (Sorting mode)* view is displayed, see [Data sources &ndash; Sorting mode](#data-sources-–-sorting-mode).
```

Ist dieses Symbol nicht in Core1 Buttons beschrieben? Evtl. hier weglassen und darauf verweisen? Das ist eine DOP Standardfunktion.

Reply:
done